extensions/cornerstone/src/init.tsx

```diff
@@ -326,6 +330,8 @@ export default async function init({
     viewportGridService.EVENTS.ACTIVE_VIEWPORT_ID_CHANGED,
     activeViewportEventListener
   );
+
+  colormaps.forEach(registerColormap);
```

Comment:
I think you should remove another registration from tmtv then? no?

Reply:
Yes good catch, removed it.